index.html

```diff
@@ -100,6 +100,10 @@ <h1 class="title" id="try">
                 <div class="smartphone"><img class="content" aria-hidden="true" src="assets/screenshots/poll.png"></div>
                 <div><a href="https://github.com/webxdc/webxdc-poll/releases/download/1.0.0/webxdc-poll.xdc">download<br><code>poll.xdc</code></a></div>
             </div>
+            <div class="is-pulled-left download">
+                <div class="smartphone"><img class="content" aria-hidden="true" src="assets/screenshots/calendar.png"></div>
+                <div><a href="https://github.com/webxdc/calendar/releases/download/0.0.1/calendar.xdc">download<br><code>calendar.xdc</code></a></div>
```

Comment:
```suggestion
                <div><a href="https://github.com/webxdc/calendar/releases/latest/download/calendar.xdc">download<br><code>calendar.xdc</code></a></div>
```

Reply:
very cool! i was not aware that this exist, we should also adapt the other urls at some point!

Reply:
Yep, I made an MR with that.